app/assets/stylesheets/application.scss

```diff
@@ -92,6 +92,12 @@ $mobileDeviceCutoff: 992px;
   }
 }
 
+.btn-secondary {
+  --bs-btn-color: #fff;
+  --bs-btn-hover-color: #fff;
+  --bs-btn-active-color: #fff;
+}
```

Comment:
Is there any other way to achieve this more cleanly?

Maybe sass extend?
https://stackoverflow.com/questions/16086377/scss-extend-a-nested-selector-and-override-the-nested-rulesets
https://sass-lang.com/documentation/at-rules/extend

Reply:
You could try if
```
.btn-secondary {
    @extend .text-white;
}
```
works

Reply:
Or have you tried removing your changes to the SCSS file? The button still stays white for me.